app/scenes/Invite.js

```diff
@@ -182,7 +192,13 @@ function Invite({ onSubmit }: Props) {
             onChange={(ev) => handleChange(ev, index)}
             value={invite.name}
             required={!!invite.email}
-            flex
+          />
+          &nbsp;
+          <InputSelectRole
+            onChange={(ev) => handleRoleChange(ev, index)}
+            value={invite.role}
+            labelHidden={index !== 0}
+            short
```

Comment:
`Send Invites` button is not aligned with the `Select`. Either we will have to increase the modal size, or we can absolutely position the button below the select.

How would you go about this?

Reply:
Lets create an issue to implement the custom select with aim of fixing this